tests/Aspire.Npgsql.EntityFrameworkCore.PostgreSQL.Tests/ConformanceTests.cs

```diff
@@ -126,28 +129,21 @@ public void DbContextCanBeAlwaysResolved()
         Assert.NotNull(dbContext);
     }
 
-    [ConditionalFact]
-    public void TracingEnablesTheRightActivitySource()
-    {
-        SkipIfCanNotConnectToServer();
+    [RequiresDockerFact]
+    public Task TracingEnablesTheRightActivitySource()
+        => RunWithFixtureAsync(obj => obj.ActivitySourceTest(key: null));
 
-        RemoteExecutor.Invoke(() => ActivitySourceTest(key: null)).Dispose();
```

Comment:
Why doesn't this test need RemoteExecutor anymore?

Reply:
IIUC, we were running these tests with RemoteExecutor to run in a separate process so changes to the db don't affect other tests. And that wasn't working because RemoteExecutor would complain about not being able to marshal `_containerFixture` field. So, I create a fresh container and run the test with that.

 But now reading the comments:
```
    // This method can have side effects (setting AppContext switch, enabling activity source by name).
    // That is why it needs to be executed in a standalone process.
    // We use RemoteExecutor for that, but it does not support abstract classes
    // (it can not determine the type to instantiate), so that is why this "test"
    // is here and derived types call it
    protected void ActivitySourceTest(string? key)
```

.. the above suggests that this would affect the running process anyway, so `RemoteExecutor` should be used. I'll fix it.